search-website/README.md

```diff
@@ -19,44 +19,51 @@ To run the sample locally, follow the steps below.
 ### Prerequisites
 
 - A GitHub account
+- A Cognitive Search resource using the `Free` pricing tier. This can be created in VS Code with the [Cognitive Search extention](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurecognitivesearch).  Copy following resources values before continuing:
+    - YOUR-SEARCH-RESOURCE-NAME
+    - YOUR-SEARCH-ADMIN-KEY - used only in bulk import for inserting
+    - YOUR-SEARCH-QUERY-KEY - used only in Function api for querying
 - [Node.js and Git](https://nodejs.org/)
 - [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=shopathome-github-jopapa) installed
-- The [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions&WT.mc_id=shopathome-github-jopapa) installed
-- The [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?WT.mc_id=shopathome-github-jopapa) installed
+    - [Static Web app extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps) installed
+    - [Cognitive Search extention](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurecognitivesearch)
 
-### Setup
+### Download sample repository
 
-1. Clone (or Fork and Clone) this repository
-1. Open `./search-website` in Visual Studio Code. 
-1. Rename the `api/local.settings.json.rename` file to `api/local.settings.json`.
+1. In a terminal, use git to clone (or Fork and Clone) this repository to your local computer:
 
-    The `local.settings.json` file holds all of the keys that the application needs.
-    
-    For this sample, there is a search index using the [goodbooks-10k dataset](https://github.com/zygmuntz/goodbooks-10k) that that is publicly available using the connection information below. The index consists of 10,000 popular books that we'll search over in our application.
-    
-    ```json
-    {
-      "IsEncrypted": false,
-      "Values": {
-        "AzureWebJobsStorage": "",
-        "FUNCTIONS_WORKER_RUNTIME": "node",
-        "SearchApiKey": "REPLACE-WITH-YOUR-SEARCH-QUERY-KEY",
-        "SearchServiceName": "REPLACE-WITH-YOUR-SEARCH-RESOURCE-NAME",
-        "SearchIndexName": "good-books",
-        "SearchFacets": "authors*,language_code"
-      }
-    }
+    ```bash
+    git clone https://github.com/Azure-Samples/azure-search-javascript-samples
     ```
 
-## Run the app locally
+1. Open that local directory in Visual Studio Code.
+
+### Run bulk import to the Search Index
+
+1. Open the `./search-website/bulk-insert/bulk_insert_books.js`.
+1. Download the [books.csv](https://raw.githubusercontent.com/zygmuntz/goodbooks-10k/master/books.csv) to the same directory as the `bulk_insert_books.js` file.
```

Comment:
when I cloned this repo, I got a _local_ books.csv file -- I don't think I have to download anything.  I downloaded the code from diberry so maybe the final version won't have this .csv file??

Reply:
That's a mistake - I need to remove that file.

Reply:
Is there a reason why we can't provide the .csv?  It stabilizes the experience.  If someone changes the dataset on the public site, our index or queries could break.